src/orders/components/OrderFulfillmentCancelDialog/OrderFulfillmentCancelDialog.tsx

```diff
@@ -15,7 +14,7 @@ import SingleAutocompleteSelectField from "@saleor/components/SingleAutocomplete
 import { OrderErrorFragment } from "@saleor/fragments/types/OrderErrorFragment";
 import { WarehouseFragment } from "@saleor/fragments/types/WarehouseFragment";
 import { buttonMessages } from "@saleor/intl";
-import { makeStyles } from "@saleor/macaw-ui";
+import { Button, makeStyles } from "@saleor/macaw-ui";
```

Comment:
Import changes, props remain the same - make sure it's intentional

Reply:
It is, we should be using only Macaw's buttons